config/globals.js

```diff
@@ -87,15 +87,6 @@ const globals = {
 	ROCKET_CHAT_ADMIN_USER: process.env.ROCKET_CHAT_ADMIN_ID,
 	ROCKET_CHAT_ADMIN_PASSWORD: process.env.ROCKET_CHAT_ADMIN_ID,
 
-	// etherpad
-	ETHERPAD_API_KEY: process.env.ETHERPAD_API_KEY,
-	ETHERPAD_API_PATH: process.env.ETHERPAD_API_PATH,
-	ETHERPAD_URI: process.env.ETHERPAD_URI,
-	ETHERPAD_OLD_PAD_URI: process.env.ETHERPAD_OLD_PAD_URI,
-	ETHERPAD_OLD_PAD_DOMAIN: process.env.ETHERPAD_OLD_PAD_DOMAIN,
-	ETHERPAD_COOKIE__EXPIRES_SECONDS: process.env.ETHERPAD_COOKIE__EXPIRES_SECONDS,
-	ETHERPAD_ETHERPAD_COOKIE_RELEASE_THRESHOLD: process.env.ETHERPAD_COOKIE_RELEASE_THRESHOLD,
-
```

Comment:
should that be out?

Reply:
yes all of it already in default.schema.json and use over hpi common. I search and found no place that use this.